scripts/generator/graphql_generator/csharp/Editor/ShopifyBuildPipeline.cs.erb

```diff
@@ -0,0 +1,61 @@
+namespace <%= namespace %>.SDK {
```

Comment:
Is there a reason why will be bundling this into the `.unitypackage` that will be distributed to users?